tests/functional/object/mcg/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,469 @@
+            tuple: (bucket_name, bucket) — the bucket in EXCEEDING_QUOTA
+                mode, ready for further assertions.
+        """
+        bucket_name = create_unique_resource_name(
+            resource_description="bucket", resource_type="quota"
+        )
+        bucket = MCGCLIBucket(bucket_name, mcg=mcg_obj, quota="2Gi")
+        logger.info(f"Created bucket {bucket_name} with 2Gi quota")
+
+        local_dir = f"/tmp/{bucket_name}"
+
+        def finalizer():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)